identity/webapp/next.config.js

```diff
@@ -33,7 +33,7 @@ const config = function () {
     assetPrefix:
       isProd && prodSubdomain
         ? `https://${prodSubdomain}.wellcomecollection.org${basePath}`
-        : '',
+        : undefined,
```

Comment:
```
warn  - Invalid next.config.js options detected:
warn  -     The value at .assetPrefix must be 1 character or more but it was 0 characters.
warn  - See more info here: https://nextjs.org/docs/messages/invalid-next-config
```

Matched the value in [our common config](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/next/next.config.js#L32)